docs/user/vulnerabilities/vulnerability/remediations/remediation/restart_required-usage.en.md

```diff
@@ -1 +1,9 @@
 # Restart Required - Usage
+
+Specifies whether a restart is required for the mitigation measure, and if so, in what scope.
+
+This is relevant as in some cases where connections exist the scope may be beyond a single machine.
+
+This information is especially relevant if the product itself is modified with the mitigating measure (workaround, mitigation, or vendor fix).
```

Comment:
Add (somewhere):
and might help the receiver of the information to make a better / risk-based decision.
May apply a none restart faster than a restart one.